src/python/bot/tasks/fuzz_task.py

```diff
@@ -1732,6 +1732,9 @@ def do_blackbox_fuzzing(self, fuzzer, fuzzer_directory, job_type):
       testcases_metadata[testcase_file_path]['gestures'] = pick_gestures(
           test_timeout)
 
+    # Prepare selecting trials in main loop below.
+    trial_selector = trials.Trials(environment)
```

Comment:
Remove environment module from argument, it should be added to import in trials.py

Reply:
Done